### PR TITLE
Add actions page items

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

- This PR adds in components which will be display on ActionsPage. Components are super dump. 

**New stuff** ✨

* ActionsPageEvent component
* ActionsPageFeedItem component

**Changes** 🏗

* Updated Permissions component to support "no padding" theme

## TODO

- Handle ActionPage layout to have correct width for events/comments list - then remove fixed width
- Make those components smarter when data will be ready [😿]

Screenshot 2020-12-02 at 12 24 45

Resolves DEV-47

### CODE REVIEW

File: src/modules/admin/components/Permissions/UserPermissions.tsx [hunk 1]
```diff
@@ -2,17 +2,23 @@ import React from 'react';
...
 interface Props {
   roles: ColonyRole[];
   directRoles: ColonyRole[];
+  appearance?: Appearance;
 }
 
 const displayName = 'admin.Permissions.UserPermissions';
 
-const UserPermissions = ({ roles, directRoles }: Props) => {
+const UserPermissions = ({ roles, directRoles, appearance }: Props) => {
```
Comment (rdig) [inline src/modules/admin/components/Permissions/UserPermissions.tsx:21]: Why is this component still in `admin`, did I forgot to mention to move it? Or maybe it was a bad rebase?

Anyway, can you please move it out of `admin` and into `dashboard` ?

File: src/modules/dashboard/components/ActionsPage/ActionsPage.tsx [hunk 2]
```diff
@@ -259,14 +266,35 @@ const ActionsPage = () => {
             />
           )}
           {!!events?.length && (
-            <ul>
+            <div style={{ width: '460px' }}>
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.tsx:269]: Wasn't this already set at 460? See here:
https://github.com/JoinColony/colonyDapp/blob/41f1217f1d93d82da32335078989d72bb412673e/src/modules/dashboard/components/ActionsPage/ActionsPage.css#L40-L43

File: src/modules/dashboard/components/ActionsPage/ActionsPageEvent/ActionsPageEvent.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+});
+
+interface Props {
+  event: ParsedEvent;
+  transactionHash?: string | null;
+  createdAt?: number;
+}
+
+const ActionsPageEvent = ({ createdAt, transactionHash }: Props) => {
+  // Mocked roles - Please make me smarter
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPageEvent/ActionsPageEvent.tsx:26]: When adding comments like this, please leave a `@TODO` doctag so we can easily find them later.

```suggestion
  // @TODO Mocked roles - Please make me smarter
```

or

```suggestion
  /*
   * @TODO Mocked roles - Please make me smarter
   */
```

File: src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  const { Decorate } = new TextDecorator({
+    username: (usernameWithAtSign) => (
+      <UserMention username={usernameWithAtSign.slice(1)} />
+    ),
+  });
+  const UserAvatar = HookedUserAvatar({ fetchUser: false });
+  return (
+    <div className={getMainClasses({}, styles, { annotation })}>
+      <div className={styles.avatar}>
+        <UserAvatar size="xs" address={walletAddress || ''} />
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx:37]: The user avatar in the specs is `28x28` while yours is `26x26`

File: src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
+
+.main {
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css:2]: I think we need a bigger margin between the feed items. I see something like `30` in the Figma file

Screenshot from 2020-12-02 16-52-54

**Review by rdig — APPROVED** [review]
Good to go. I found minor stuff here and there which I've marked, but nothing major.

After you fixed them, you can merge this, so I can pull your changes into #2037 and wire up the comments.

:tada:

File: src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css (hunk 5 above)
Comment (alicjakujawa) [reply src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css:2]: Screenshot 2020-12-02 at 17 10 02
Depends on which design we're looking for 🤦  23-16px of padding, it means something like 7-8px of margin, i can add it

File: src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx (hunk 4 above)
Comment (alicjakujawa) [reply src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx:37]: But we don't have 28x28 avatars, till now we was using only options from Avatar appearance object to be consistent, do we want change it?

Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.tsx:37]: I know we don,t but sometimes the Figma files have non-standard sizes. 

You can handle this in one of three ways:
- Talk to Karol about the discrepancy and see how this came about
- Just change the size locally in `ActionsPageFeedItem` and overwrite the `xs` size
- Leave it like it is as the size different is imperceptible

I would go with the latter as is the quickest. 

We'll most likely have some more time when wiring this, and we'll do a couple of polishing runs before launch

File: src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css (hunk 5 above)
Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css:2]: Yes, I know, the Figma file doesn't really give proper specs in this regard. 

I just like bigger margins visually :P

I think you can leave them as they currently are.

**Pull request merged**